src/main/webapp/index.html

```diff
@@ -10,12 +45,11 @@
   <body>
     <header>
       <h1 class="logo">smallcity</h1>
-      <button class="geolocate-button" onclick="getGeolocation()">Locate Me</button>
+      <button class="geolocate-button" onclick="getGeolocation()"><i class="fas fa-location-arrow"></i> locate me</button>
```

Comment:
FWIW this likely could have been done using pure css, rather than including the new library.